### PR TITLE
feat: 🎸 [JIRA:HCPSDKFIORIUIKIT-2717] Feedback Patterns

### PR DESCRIPTION
**Pull request by hengyi-zhang**

### CODE REVIEW

File: Apps/Examples/Examples/en.lproj/Localizable.strings
Comment (xiaoqinggrace) [inline Apps/Examples/Examples/en.lproj/Localizable.strings:1]: The sample code is for the application team's reference. Localization is not required.

File: Apps/Examples/Examples/zh-Hans.lproj/Localizable.strings
Comment (xiaoqinggrace) [inline Apps/Examples/Examples/zh-Hans.lproj/Localizable.strings:1]: The translation is provided by another team.

File: Sources/FioriSwiftUICore/_localization/zh-Hans.lproj/FioriSwiftUICore.strings
Comment (xiaoqinggrace) [inline Sources/FioriSwiftUICore/_localization/zh-Hans.lproj/FioriSwiftUICore.strings:1]: We do not need to add translation. There will be a translation process for product release.

**Comment by xiaoqinggrace** [comment]
UI components jump unexpectedly when deleting an entry on iPad. It can be fixed in a separate PR.

File: Apps/Examples/Examples/en.lproj/Localizable.strings
Comment (hengyi-zhang) [reply Apps/Examples/Examples/en.lproj/Localizable.strings:1]: fixed

File: Apps/Examples/Examples/zh-Hans.lproj/Localizable.strings
Comment (hengyi-zhang) [reply Apps/Examples/Examples/zh-Hans.lproj/Localizable.strings:1]: fixed!

File: Sources/FioriSwiftUICore/_localization/zh-Hans.lproj/FioriSwiftUICore.strings
Comment (hengyi-zhang) [reply Sources/FioriSwiftUICore/_localization/zh-Hans.lproj/FioriSwiftUICore.strings:1]: fixed!

**Review by xiaoqinggrace — APPROVED** [review]

**Pull request merged**